php/libraries/SinglePointLogin.class.inc

```diff
@@ -318,7 +318,8 @@ class SinglePointLogin
                 }
 
                 if ($row['Active'] == 'N'
-                    || $this->disabledDueToInactivity($username, 365)
+                    || ($this->disabledDueToInactivity($username, 365)
+                        && $row['Active'] != 'Y')
```

Comment:
This will not work, even if you change Active to Y in the DB, the user will be shown the message below (line 324-235)

Reply:
I do believe that something to a larger scale, like adding a parameter in the DB `inactiveOverride` or change the process to disable the user after a period without login in.